.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 jobs:
-  step-one:
+  step-two:
```

Comment:
This should be `step-one` for this branch 👍

Reply:
Made it!